tools/convert-help-center-docs-to-mdx

```diff
@@ -607,6 +607,17 @@ def get_include_files_info(include_input_dir: str) -> dict[str, IncludeFileInfo]
             include_files_info[name] = {
                 "is_only_ordered_list": is_include_only_ordered_list(markdown_string)
             }
+            # We are just checking one level deep and not checking for
+            # any recursions, which would be fine for our use case.
```

Comment:
Looks like we have some that are more than one level deep, e.g., the user roles article:
`user-roles.md` -> `{!manage-this-user.md!}` -> `{!user-card-three-dot-menu.md!}` -> `{!right-sidebar-user-card.md!}`